composer.json

```diff
@@ -5,24 +5,23 @@
         "MIT"
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
-        "doctrine/orm": "^2.14.0",
-        "shipmonk/doctrine-hint-driven-sql-walker": "^1.0.0"
+        "php": "^8.1",
+        "doctrine/orm": "^3.0.0",
```

Comment:
Isn't possible to provide `^2 || ^3` support at least for some versions in order to give a smoother upgrade path ?

Reply:
Theoretically possible, but since we use very narrow part of Doctrine API, it should not be hard to upgrade both doctrine/orm and this package at the same time.